packages/engine/Source/Scene/CameraEventAggregator.js

```diff
@@ -141,21 +144,29 @@ function listenToWheel(aggregator, modifier) {
     movement = aggregator._movement[key] = {};
   }
 
+  let lastMovement = aggregator._lastMovement[key];
+  if (!defined(lastMovement)) {
+    lastMovement = aggregator._lastMovement[key] = {
+      startPosition: new Cartesian2(),
+      endPosition: new Cartesian2(),
+      valid: false,
+    };
+  }
+
   movement.startPosition = new Cartesian2();
+  Cartesian2.clone(Cartesian2.ZERO, movement.startPosition);
   movement.endPosition = new Cartesian2();
 
   aggregator._eventHandler.setInputAction(
     function (delta) {
       // TODO: magic numbers
-      const arcLength = 15.0 * CesiumMath.toRadians(delta);
-      if (!update[key]) {
-        movement.endPosition.y = movement.endPosition.y + arcLength;
-      } else {
-        Cartesian2.clone(Cartesian2.ZERO, movement.startPosition);
-        movement.endPosition.x = 0.0;
-        movement.endPosition.y = arcLength;
-        update[key] = false;
-      }
+      const arcLength = 7.5 * CesiumMath.toRadians(delta);
```

Comment:
> I have already halfing delta value by calculate the arcLength

@jiangheng90 I see. I think we should further reduce this since the zoom is currently fast and its easy to loose control. 

If you could, please also remove the comment that says `TODO: magic numbers`. We should not have left the `TODO` comment in the code to begin with.

Reply:
@ggetz already removed TODO comment. So. I see may TODO comment in cesium. But I'm not sure in which case this will be removed